src/view/plandetail/PlaceInfoTab.tsx

```diff
@@ -125,14 +127,22 @@ export const TabPanel = ({
 const TabPanelInformation = ({
     categories,
     priceRange,
+    googlePlaceReviews,
 }: {
     categories: PlaceCategory[];
     priceRange: PriceRange | null;
+    googlePlaceReviews: GooglePlaceReview[] | null;
 }) => {
     const isCategoryEmpty = categories.length === 0;
     const isPriceRangeEmpty = !priceRange || priceRange.max === 0;
+    const reviews = googlePlaceReviews;
+    const isGooglePlaceReviews = !reviews || reviews.length === 0;
```

Comment:
[refactor] ここは上の例に従っておくと良いと思います
この変数は下のif文で使われているように、「何も情報がなかった」場合に用いられるので。
```suggestion
    const isGooglePlaceReviewEmpty = !reviews || reviews.length === 0;
```